<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeroomMainWindow</class>
 <widget class="QMainWindow" name="TimeroomMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Timeroom</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="dirView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Read"/>
   <addaction name="action_Initialize"/>
   <addaction name="action_Process_Curves"/>
   <addaction name="action_Process_Gradients"/>
   <addaction name="action_Process"/>
   <addaction name="action_Save"/>
   <addaction name="action_Exit"/>
   <addaction name="action_About"/>
  </widget>
  <action name="action_Initialize">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Initialize</string>
   </property>
   <property name="toolTip">
    <string>Auto-detect keyframes based on EV changes (discards existing ratings)</string>
   </property>
  </action>
  <action name="action_Read">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Read</string>
   </property>
   <property name="toolTip">
    <string>Read (or re-read) sidecar files, discarding any unsaved changes</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="action_Process">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Process</string>
   </property>
   <property name="toolTip">
    <string>Interpolate values between keyframes</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="iconText">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit without saving changes</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About Timeroom</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Write outstanding changes (ratings or interpolated settings) to sidecars</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="action_Process_Curves">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Interpolate Curves</string>
   </property>
   <property name="toolTip">
    <string>Whether to interpolate tone curves during process</string>
   </property>
  </action>
  <action name="action_Process_Gradients">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Interpolate Gradients</string>
   </property>
   <property name="toolTip">
    <string>Whether to interpolate gradients during process</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
